core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -198,7 +198,7 @@ trait Endpoint[A] { self =>
       final def apply(input: Input): Endpoint.Result[pa.Out] = inner(input)
 
       override def item = items.MultipleItems
-      final override def toString: String = s"${other.toString} :: ${self.toString}"
+      final override def toString: String = s"${other.toString}/${self.toString}"
```

Comment:
I wonder if we should also alter the `coproduct` to-string. I'd imagine, something like `\n` (a new line) would be quite human-readable. What do you think?

Reply:
Hm, little imagination
route - `root :: (("admin" :: header :: (addGoods :+: ...)) :+: (("goods" :: id) :+: ("cart" :: cookies) :+: "about" :+: ...))`
string representation - 
```
"root" / ( 
    "admin" / header / ( addGoods
    ... 
)
(
    ( "goods" / id)
    ( "cart" / cookies)
    "about"
    ...
)
```
I'm not sure it's somehow readable.

Reply:
@vkostyukov I'm sorry, I wasn't paying attention and I wrote "coproduct" in the commit and PR description, where I obviously meant "product" (or "composition"). I edited the PR description and force pushed the amended commit with a proper description (I used the --force-with-lease flag which stops the push in case someone else already pushed something to the remote).

Yes, I think newline could work for coproduct. I feel like @optician tried to make a point of how clumsy such string representation would be, but I actually like it.